runtime/src/comm/ugni/comm-ugni.c

```diff
@@ -962,17 +966,14 @@ typedef struct {
   c_sublocid_t  subloc;  // target sublocale
   unsigned char fast:          1;
   unsigned char blocking:      1;
-  unsigned char payload_size;
+  unsigned short payload_size;
```

Comment:
`uint_least16_t` might be preferable from a descriptive point of view, though the end result in terms of type size would be the same on Cray XE/XC, where this is used.

Reply:
Unless you feel strongly about this, I would prefer to leave this as-is. I think we want to keep this at 2 bytes (and not at least 2 bytes) and I don't see any particular value in changing to `uint_*t` when the other members don't use that style.

Reply:
I don't feel that strongly about it, and I know that my previous lack of adherence to this doesn't give me much of a leg to stand on.  :-)